### PR TITLE
chore(benchmarks): add cloud benchmark setup

### PR DESCRIPTION
**Pull request by ChrisKujawa**
## Description

 * Add description on how to run a benchmark with the camunda cloud cluster
 * Add new script to create new cloud benchmark folders
 * A Cloud benchmark folder contains:

    *  a secret deployment which contains cloud credentials. This allows to only edit one file and store the credentials secretly.
    * starter, worker, timer and simpleStarter all using the secret
    * Makefile to deploy the resources

## Definition of Done

_Not all items need to be done depending on the issue and the pull request._

Code changes:
* [ ] The changes are backwards compatibility with previous versions
* [ ] If it fixes a bug then PRs are created to backport the fix to the last two minor versions

Testing:
* [ ] There are unit/integration tests that verify all acceptance criterias of the issue
* [ ] New tests are written to ensure backwards compatibility with further versions
* [ ] The behavior is tested manually
* [ ] The impact of the changes is verified by a benchmark 

Documentation: 
* [ ] The documentation is updated (e.g. BPMN reference, configuration, examples, get-started guides, etc.)
* [ ] New content is added to the release announcement

### CODE REVIEW

**Comment by Zelldon** [comment]
i added you both @saig0 and @pihme so you're aware of it as well

**Comment by Zelldon** [comment]
@npepinpe do you have time to look at this? otherwise I can also remove you from the review

File: benchmarks/setup/cloud-default/cloudcredentials.yaml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: cloud-credentials
+type: Opaque
+stringData:
+  clientId: hH55UFivfw-bbHAuPwN545oyv8tTdW0z
```
Comment (npepinpe) [inline benchmarks/setup/cloud-default/cloudcredentials.yaml:7]: Are these actual credentials?

**Review by npepinpe — CHANGES_REQUESTED** [review]
Generally looks good, but are those actual secrets in the default one, or just dummy ones that get replaced?

File: benchmarks/setup/cloud-default/cloudcredentials.yaml (hunk 1 above)
Comment (Zelldon) [reply benchmarks/setup/cloud-default/cloudcredentials.yaml:7]: Example ones; I already deleted the client :wink: Wanted to make the example as real as possible otherwise it is often confusing where to put and how the values.

Comment (npepinpe) [reply benchmarks/setup/cloud-default/cloudcredentials.yaml:7]: Can we add a comment on the README to make sure nobody actually commits real credentials :sweat_smile: ?

Comment (Zelldon) [reply benchmarks/setup/cloud-default/cloudcredentials.yaml:7]: Of course we can :D I think nobody does that, but even when it is just a small scope right :D

Comment (npepinpe) [reply benchmarks/setup/cloud-default/cloudcredentials.yaml:7]: I'm thinking, maybe someone sees the example and thinks "oh it's safe to commit them I guess"? :man_shrugging:

Comment (Zelldon) [reply benchmarks/setup/cloud-default/cloudcredentials.yaml:7]: ok done

Comment (Zelldon) [reply benchmarks/setup/cloud-default/cloudcredentials.yaml:7]: ah ok missed your comment :D should I add this comment to the default secret as well ?

Comment (npepinpe) [reply benchmarks/setup/cloud-default/cloudcredentials.yaml:7]: Hm, or can we .gitignore it? I think if the original is already committed then it won't be removed, but no further ones would be committed?

Comment (Zelldon) [reply benchmarks/setup/cloud-default/cloudcredentials.yaml:7]: Ok lets try that

Comment (Zelldon) [reply benchmarks/setup/cloud-default/cloudcredentials.yaml:7]: ok works

**Review by npepinpe — APPROVED** [review]
Thanks :+1:

File: benchmarks/setup/README.md [hunk 2]
```diff
@@ -71,3 +71,23 @@ In order to do this easily, just run:
...
+
+## How to run a Benchmark in Camunda Cloud
+
+We currently waiting until https://github.com/camunda-cloud/enhancements/blob/master/CEP017-programmatic-access-to-console.md is implemented, until this is done we need to do some steps manually.
+
+### Setup Cloud Cluster
+
+ * Go to https://console.cloud.ultrawombat.com
+ * Login with your credentials, reach out to cloud team if you haven't one
+ * Create a new cluster, as region choose 'new chaos'. This means the cluster will be created in 'ultrachaos' gke cluster
```
Comment (pihme) [inline benchmarks/setup/README.md:83]: Who is this document targeted at? I guess customers/community members don't have access to this region. So I guess this is targeted internally. I think it makes sense to make this more explicit.

**Review by pihme — APPROVED** [review]

File: benchmarks/setup/README.md (hunk 2 above)
Comment (Zelldon) [reply benchmarks/setup/README.md:83]: I can add something like this:


_This targets only internal use cases and is not available for the outside._

Comment (Zelldon) [reply benchmarks/setup/README.md:83]: @pihme like this?

Comment (pihme) [reply benchmarks/setup/README.md:83]: Yes, but maybe rephrase _outside_ to _external parties_

Comment (Zelldon) [reply benchmarks/setup/README.md:83]: ok thanks
